src/components/NavBar/Navbar.tsx

```diff
@@ -64,6 +64,9 @@ const PageTabs = () => {
 }
 
 const Navbar = () => {
+  const { pathname } = useLocation()
+  const isNftPage = pathname.startsWith('/nfts')
```

Comment:
Can you rebase this PR after merging the nfts fix to verify that the TagIcon shows as expected for Sell

Reply:
I merged latest locally and was seeing the shopping bag, not the tag for sell

Reply:
yes